### PR TITLE
Bump ui to 1.11+v1.16.0

### PR DESCRIPTION
**Pull request by nLight**
## High-level description

Updated to [DC/OS UI 1.11+v1.16.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.16.0/CHANGELOG.md)

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-3801](https://jira.mesosphere.com/browse/DCOS_OSS-3801) - Support docker parameters in Metronome
  - [DCOS-39662](https://jira.mesosphere.com/browse/DCOS-39662) - Refactor ServiceDebugContainer

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by nLight** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by nLight** [comment]
@mesosphere-mergebot bump-ee

File: CHANGES.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ### Notable changes
 
 * Updated REX-Ray version to 0.11.2 (DCOS_OSS-3597) [rexray v0.11.2](https://github.com/rexray/rexray/releases/tag/v0.11.2)
-* Updated to [DC/OS UI 1.11+v1.15.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.15.0/CHANGELOG.md)
+* Updated to [DC/OS UI 1.11+v1.16.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.16.0/CHANGELOG.md)
```
Comment (juliangieseke) [inline CHANGES.md]: IMHO we should add a new line here instead of replacing the old one 🤔

Comment (nLight) [reply CHANGES.md]: Why?

Comment (juliangieseke) [reply CHANGES.md]: when we update ui multiple times, we need to keep track of all the updates 🤷‍♂️

Comment (orlandohohmeier) [reply CHANGES.md]: 🎨  I'd change the line to `Updated DC/OS UI to [1.11+v1.16.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.16.0/CHANGELOG.md)`.

**Comment by nLight** [comment]
@orlandohohmeier do you know where do I find `mergebot/enterprise/build-status/aggregate` so that I could troubleshoot it?

**Comment by nLight** [comment]
@mesosphere-mergebot label ship it

**Comment by juliangieseke** [comment]
needs another approval… cc @orlandohohmeier

**Comment by juliangieseke** [comment]
wait… it has ship it already? I dont get it anymore 😕

**Comment by nLight** [comment]
@juliangieseke it had 2 reviews

**Comment by nLight** [comment]
@orsenthil that one got messed up. EE got merged with the previous train. Would you please help me troubleshooting it?

**Comment by orsenthil** [comment]
@nLight  - Let's just merge these https://github.com/dcos/dcos/pull/3199 | https://github.com/mesosphere/dcos-enterprise/pull/3164 then - The this change will go In and the EE reference will be updated to the correct HEAD of 1.11.

**Comment by nLight** [comment]
@orsenthil added my ✅ \o/

**Pull request merged**